Vegeting/Screens/RegisterUserProfile/UserInterestViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  UserInterestViewController.swift
+//  Vegeting
+//
+//  Created by 최동권 on 2022/11/06.
+//
+
+import UIKit
+
+class UserInterestViewController: UIViewController {
+
+    private let progressBarImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "progress5")
+        return imageView
+    }()
+    
+    private let titleLabel: UILabel = {
+       let label = UILabel()
+        label.font = .preferredFont(forTextStyle: .title3, compatibleWith: .init(legibilityWeight: .bold))
+        label.text = "관심있는 모임을 선택해주세요. (최대 3개)"
+        return label
+    }()
+        
+    private lazy var profileRegisterButton: BottomButton = {
+       let button = BottomButton()
+        button.setTitle("프로필 설정 완료", for: .normal)
+        button.isEnabled = false
+        button.addTarget(self, action: #selector(profileRegisterButtonTapped), for: .touchUpInside)
+        return button
+    }()
+    
+    private let interestList: [String] = ["맛집", "카페", "행사", "패션", "뷰티", "환경", "정치", "친목", "동물권", "요리", "베이킹"]
+    private lazy var selectInterestView = InterestView(interestList: interestList, entryPoint: .register)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupDelegate()
+        setupLayout()
+        configureUI()
+    }
+    
+    private func setupDelegate() {
+        selectInterestView.delegate = self
+    }
+    
+    private func setupLayout() {
+        view.addSubviews(progressBarImageView,
+                         titleLabel,
+                         selectInterestView,
+                         profileRegisterButton)
+        
+        progressBarImageView.constraint(top: view.topAnchor,
+                                        leading: view.leadingAnchor,
+                                        padding: UIEdgeInsets(top: 116, left: 20, bottom: 0, right: 0))
+        progressBarImageView.constraint(.widthAnchor, constant: 186)
+        progressBarImageView.constraint(.heightAnchor, constant: 26)
+        
+        titleLabel.constraint(top: progressBarImageView.bottomAnchor,
+                              leading: view.leadingAnchor,
+                              padding: UIEdgeInsets(top: 38, left: 20, bottom: 0, right: 0))
+        
+        selectInterestView.constraint(top: titleLabel.bottomAnchor,
+                                      leading: view.leadingAnchor,
+                                      trailing: view.trailingAnchor,
+                                      padding: UIEdgeInsets(top: 11, left: 20, bottom: 0, right: 20))
```

Comment:
p2; 
앞서 설명드렸듯, 한 뷰에 height로 고정되는 것보단 최하단 버튼의 topAnchor를 기준으로 레이아웃을 잡으면 더욱 좋을것 같습니다.
```suggestion
selectInterestView.constraint(top: titleLabel.bottomAnchor,
                                      leading: view.leadingAnchor,
                                      bottom: profileRegisterButton.topAnchor,
                                      trailing: view.trailingAnchor,
                                      padding: UIEdgeInsets(top: 11, left: 20, bottom: 0, right: 20))
```

Reply:
https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/143/commits/bd3b3b1c8b5b30c85530216c50174519e7eb7fd9#diff-89d6cca1f922df5bb84a005179dc6e5226607c666cb6d1237b4e97ec87a2458bR66
에서 반영완료!